pkg/cmd/server/api/v1/conversions.go

```diff
@@ -17,6 +17,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.APILevels) == 0 {
 				obj.APILevels = internal.DefaultOpenShiftAPILevels
 			}
+			if len(obj.JenkinsPipelineConfig.Namespace) == 0 {
+				obj.JenkinsPipelineConfig.Namespace = "openshift"
+			}
+			if len(obj.JenkinsPipelineConfig.TemplateName) == 0 {
+				obj.JenkinsPipelineConfig.TemplateName = "jenkins"
+			}
+			if len(obj.JenkinsPipelineConfig.ServiceName) == 0 {
+				obj.JenkinsPipelineConfig.ServiceName = "jenkins-ci"
```

Comment:
`jenkins-pipeline` or `jenkins-ci` sound ok.  Remember that this is going to be part of the route name, so what route do we want on the outside for this?

`ci-namespace.myapps.com`?
`jenkins-ci-namespace.myapps.com`?

I'd probably prefer to go shorter rather than longer - we've been discussing having the git server take `git` in the namespace.